composer.json

```
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "~4",
+        "silverstripe/framework": "~4"
+    },
+    "suggest": {
         "silverstripe/asset-admin": "~1"
```

```suggestion
        "silverstripe/asset-admin": "Provides an admin interface for managing your Silverstripe assets"
```

The suggest schema is key => value: module => description

---

Right, good point. I think I might rewrite it to say "Necessary to make use of bulk-upload features." ie to explain why we're suggesting hat module.